lib/node/rpc.js

```diff
@@ -256,7 +256,7 @@ class RPC extends RPCBase {
 
   async help(args, _help) {
     if (args.length === 0)
-      return 'Select a command.';
+      return `Select a command:\n  ${Object.keys(this.calls).join('\n')}`;
```

Comment:
I think you can remove the spaces: `Select a command:\n${Object.keys(this.calls).join('\n')}`
Testing a0803bd0953233606ad03fdf7dedee2e10995e83 the output looks a little awkward right at the top:
```
$ bcoin-cli --network=regtest rpc
Select a command:
  stop
help
getblockchaininfo
getbestblockhash
getblockcount
...
```